internal/controllers/clusterextension_admission_test.go

```diff
@@ -186,6 +190,52 @@ func TestClusterExtensionAdmissionChannel(t *testing.T) {
 	}
 }
 
+func TestClusterExtensionAdmissionInstallNamespace(t *testing.T) {
+	tooLongError := "spec.installNamespace: Too long: may not be longer than 63"
+	regexMismatchError := "spec.installNamespace in body should match"
+
+	testCases := []struct {
+		name             string
+		installNamespace string
+		errMsg           string
+	}{
+		{"just alphanumeric", "justalphanumberic1", ""},
+		{"hypen-separated", "hyphenated-name", ""},
+		{"no install namespace", "", regexMismatchError},
+		{"dot-separated", "dotted.name", regexMismatchError},
+		{"includes version", "channel-has-version-1.0.1", regexMismatchError},
```

Comment:
We already check for dot separated, I don't think we need to explicitly check for "includes version" for install namespaces

```suggestion
```

Reply:
Yep good point, this was a copy/paste job from the channel name tests, and this one does not seem relevant here. I'll remove.